apix-impl/51/src/main/java/eu/xenit/apix/alfresco51/CommentServiceImpl51.java

```diff
@@ -0,0 +1,94 @@
+package eu.xenit.apix.alfresco51;
+
+import com.github.dynamicextensionsalfresco.osgi.OsgiService;
+import eu.xenit.apix.alfresco.ApixToAlfrescoConversion;
+import eu.xenit.apix.alfresco.comments.CommentService;
+import eu.xenit.apix.comments.Comment;
+import eu.xenit.apix.content.IContentService;
+import eu.xenit.apix.node.INodeService;
+import eu.xenit.apix.permissions.IPermissionService;
+import java.io.Serializable;
+import java.util.Set;
+import org.alfresco.model.ContentModel;
+import org.alfresco.repo.security.authentication.AuthenticationUtil;
+import org.alfresco.repo.site.SiteModel;
+import org.alfresco.service.ServiceRegistry;
+import org.alfresco.service.cmr.lock.LockService;
+import org.alfresco.service.cmr.lock.LockStatus;
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.cmr.repository.NodeService;
+import org.alfresco.service.cmr.security.AccessStatus;
+import org.alfresco.service.cmr.security.PermissionService;
+import org.alfresco.service.namespace.QName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service("eu.xenit.apix.comments.CommentService")
+@OsgiService
+public class CommentServiceImpl51 extends CommentService {
+
+    private static final Logger log = LoggerFactory.getLogger(CommentServiceImpl51.class);
+
+    protected ServiceRegistry serviceRegistry;
+
+    @Autowired
+    public CommentServiceImpl51(
+            ServiceRegistry serviceRegistry,
```

Comment:
Instead of injecting `ServiceRegistry`, we usually inject the necessary service itself